static/less/auth.less

```diff
@@ -142,131 +143,15 @@ section.org-login {
   }
 }
 
-// 2FA
-
-div.u2f-box {
```

Comment:
Pretty much replaced all this with our existing components + some small styled components.